app/models/region.rb

```diff
@@ -2,4 +2,14 @@ class Region < ApplicationRecord
   has_many :subprojects
 
   validates :name, presence: true, uniqueness: true
+  validates :latitude,
```

Comment:
See my other comment below, but do we want to add tests for this and if so where (model or controller tests). Since we test uniqueness via a controller request.